src/components/Home.tsx

```diff
@@ -1,12 +1,15 @@
-import { Link } from "react-router-dom";
+import { Heading, Link } from "@chakra-ui/react";
+import { Link as RouterLink } from "react-router-dom";
 
 const Home = () => {
-    return (
-        <> 
-            <h1>I am home</h1>
-            <Link to="/">BACK</Link>
-        </>
-    )
-}
+  return (
+    <>
+      <Heading>I am home</Heading>
+      <Link as={RouterLink} to="/">
```

Comment:
This is from Chakra's documentation: https://chakra-ui.com/docs/components/link#usage-with-routing-library

I'm using a Chakra `Link` component but it's going to be a `react-router-dom` link that is styled and used here.